pegen/grammar.py

```diff
@@ -72,6 +72,9 @@ def __init__(self, name: str, type: Optional[str], rhs: Rhs):
     def is_loop(self) -> bool:
         return self.name.startswith("_loop")
 
+    def is_node_with_sep(self) -> bool:
```

Comment:
I'm not excited about the long name here (esp. since it's repeated in a keyword arg later).